source/common/access_log/access_log_formatter.cc

```diff
@@ -221,6 +223,18 @@ RequestInfoFormatter::RequestInfoFormatter(const std::string& field_name) {
     field_extractor_ = [](const RequestInfo::RequestInfo& request_info) {
       return AccessLogFormatUtils::durationToString(request_info.firstUpstreamRxByteReceived());
     };
+  } else if (field_name == "RESPONSE_TX_DURATION") {
```

Comment:
I think this needs a doc update and a release note? https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log?

Reply:
@mattklein123 `%RESPONSE_DURATION% isn't public either... so my thinking was lets have it be private for now and then promote it as publicly available interface/variable? I am happy either way though.

Reply:
What do you mean not public? I think if it's not documented that's a doc bug. Can you fix both? :)

Reply:
Haha, sure — diff coming up.